packages/@aws-cdk/cdk/lib/cloudformation/stack.ts

```diff
@@ -101,6 +96,11 @@ export class Stack extends Construct {
    */
   public readonly name: string;
 
+  /**
+   * Used to determine if this construct is a stack.
+   */
+  protected readonly _isStack = true;
```

Comment:
Why protected and not private?

Reply:
Private gives you an "unused private member" static typechecker error.

I guess I could override it but this seems fine as well.

Reply:
Makes sense